lib/plugins/audit.js

```diff
@@ -91,6 +95,7 @@ function auditLogger(options) {
 
     function audit(req, res, route, err) {
         var latency = res.get('Response-Time');
+        var timestamp = new Date().toString();
```

Comment:
Don't use the fully qualified date object here, this will make it hard to parse. Instead, give it ns since epoch